Floofbot/Services/EventLoggerService.cs

```diff
@@ -608,7 +657,35 @@ class WordFilterService
             List<FilteredWord> _filteredWords;
             DateTime _lastRefreshedTime;
 
-            bool hasFilteredWord(FloofDataContext floofDb, string messageContent, ulong serverId, ulong channelId)
+            public bool hasFilteredWord(FloofDataContext floofDb, string messageContent, ulong serverId) // names
```

Comment:
It's nice that the comment's there, otherwise it'd be much harder to tell when to call which overload.